src/ice_match/models/__init__.py

```
@@ -4,4 +4,4 @@
 from .match_result import MatchResult, MatchType
 from .recon_status import ReconStatus, AggregationType
 
-__all__ = ["Trade", "TradeSource", "MatchResult", "MatchType", "ReconStatus", "AggregationType"]
\ No newline at end of file
+__all__ = ["Trade", "TradeSource", "MatchResult", "MatchType", "ReconStatus", "AggregationType"]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)